src/snapred/backend/recipe/algorithm/LiteDataCreationAlgo.py

```diff
@@ -4,94 +4,89 @@
+        self.rawDataPath: str = self.IPTS + "nexus/SNAP_{}.nxs.h5".format(self.runNumber)
+        pass
+
     def PyExec(self):
         self.log().notice("Lite Data Creation START!")
-        # define super pixel dimensions
-        superPixel = [8, 8]
 
-        # get output workspace name
+        # load input workspace
+        inputWorkspaceName = self.getProperty("InputWorkspace").value
+
+        # load run number
+        run = RunConfig(**json.loads(self.getProperty("Run").value))
+        self.chopIngredients(run)
+
+        # create outputworkspace
         outputWorkspaceName = self.getProperty("OutputWorkspace").value
 
-        # load input workspace
-        inputWorkspace = self.getProperty("InputWorkspace").value
+        # load file
+        self.mantidSnapper.LoadEventNexus(
+            "Loading Event Nexus for {}...".format(self.rawDataPath),
+            Filename=self.rawDataPath,
+            OutputWorkspace=inputWorkspaceName,
+            NumberOfBins=1,
+            LoadMonitors=False,
+        )
 
-        # clone the workspace
-        self.mantidSnapper.CloneWorkspace(
-            "Cloning input workspace for lite data creation...",
-            InputWorkspace=inputWorkspace,
-            OutputWorkspace=outputWorkspaceName,
+        groupingWorkspaceName = f"{self.runNumber}_lite_grouping_ws"
+
+        # load grouping map
+        self.mantidSnapper.LoadGroupingDefinition(
+            "Loading lite grouping information...",
+            GroupingFilename=str(Config["instrument.lite.map.file"]),
+            InstrumentFilename=str(Config["instrument.native.definition.file"]),
+            OutputWorkspace=groupingWorkspaceName,
         )
+
         self.mantidSnapper.executeQueue()
-        outputWorkspace = self.mantidSnapper.mtd[outputWorkspaceName]
-
-        # IEventWorkspace ID is equivalent to spec ID
-        numSpec = outputWorkspace.getNumberHistograms()
-
-        # array to hold spectrum data
-        spectrumData = []
-
-        # iterate over IEventWorkspace IDs
-        for spec in range(numSpec):
-            # get event list per workspace
-            spectrum = outputWorkspace.getSpectrum(workspaceIndex=spec)
-            spectrumData.append(spectrum)
-
-        # allocate array of event IDs
-        for spectrum in spectrumData:
-            eventIDs = spectrum.getDetectorIDs()
-            for event in eventIDs:
-                superID = self.getSuperID(nativeID=event, xdim=superPixel[0], ydim=superPixel[1])
-
-                # clear detector IDs and then add the new modified super ID
-                spectrum.clearDetectorIDs()
-                spectrum.addDetectorID(superID)
-
-        self.mantidSnapper.LoadInstrument(
-            "Loading instrument...",
-            Workspace=outputWorkspace,
-            Filename=Resource.getPath("/inputs/pixel_grouping/SNAPLite_Definition.xml"),
-            RewriteSpectraMap=False,
+        self.mantidSnapper.mtd[groupingWorkspaceName]
```

Comment:
The linter "fixed" this line by removing its assignment.  You can just delete the line.